iiwa_driver/src/iiwa.cpp

```diff
@@ -321,23 +322,23 @@ namespace iiwa_ros {
 
         for (int i = 0; i < _num_joints; i++) {
             _joint_position[i] = _robot_state.getMeasuredJointPosition()[i];
-            _joint_velocity[i] = filters::exponentialSmoothing((_joint_position[i] - _joint_position_prev[i]) / elapsed_time.toSec(), _joint_velocity[i], 0.2);
+            _joint_velocity[i] = filters::exponentialSmoothing((_joint_position[i] - _joint_position_prev[i]) / ctrl_duration.toSec(), _joint_velocity[i], 0.2);
```

Comment:
Why not `elapsed_time` here? I guess you have good reasons, but I'd like to have them documented.

Reply:
well, one of them is that we should handle interruptions then. That leads to very large values for the measured elapsed time. Like seconds.

The second reason is that when we implemented the controller for the IROS 2021 paper, you told me that we should not use the actual time when integrating, but only the expected time. I applied the same reasoning on the derivative here.

Reply:
Good enough! Merging... Thanks again! :)